app/src/main/java/com/devshish/internship/presentation/ui/MainActivity.kt

```diff
@@ -10,8 +10,10 @@ import androidx.navigation.NavController
 import androidx.navigation.fragment.NavHostFragment
```

Comment:
https://stackoverflow.com/questions/68473542/mediasessioncompattargeting-s-version-31-and-above-requires-that-one-of-flag

![image](https://user-images.githubusercontent.com/23121846/140087834-e2297d29-5f91-40f7-a374-945a4c1d846f.png)

https://developer.android.com/jetpack/androidx/releases/media#version_141_3

Use this - and because of this it will work on Android 12